includes/class-wp-job-manager.php

```diff
@@ -597,4 +603,15 @@ public function include_admin_files() {
 			include_once JOB_MANAGER_PLUGIN_DIR . '/includes/admin/class-wp-job-manager-admin.php';
 		}
 	}
+
+	/**
+	 * Sets a transient to track the activation time of WP Job Manager plugin.
+	 *
+	 * @return void
+	 *
+	 * @since $$next-version$$
+	 */
+	public function set_activation_time() {
+		set_transient( 'job_manager_activation_time', time() );
```

Comment:
Transients should be thought of as a cache that can be deleted any time. In our case this won't be a big issue but I think that having an option for that will be better.